src/main/java/Duke.java

```diff
@@ -1,10 +1,10 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Controller bot = new Controller();
+        bot.greet();
+        while(true){
+            bot.listen();
+        }
```

Comment:
I like the idea of having the controller.java in a separate file and not a whole chunk of code in the main program! Very Nice!